src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -58,7 +58,7 @@ public override StartCommandResult OnStartCommand([NotNull] Intent? intent, Star
 			{
 				foreach (var exception in t.Exception.InnerExceptions)
 				{
-					System.Diagnostics.Trace.WriteLine($"[error] {exception}, {exception.Message}");
+					System.Diagnostics.Trace.TraceError($"[error] {exception}, {exception.Message}");
```

Comment:
I know you haven't changed this but won't Exception.ToString include the Message? So this would result in the message being reported twice